sysutils/lima/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/lima-vm/lima 1.0.7 v
+go.setup            github.com/lima-vm/lima 1.1.0 v
```

Comment:
now 1.1.1

Reply:
I don't think that MacPorts is affected by https://github.com/lima-vm/lima/pull/3566 (and thus we don't _need_ 1.1.1 urgently), but I can open a PR for it a bit later.